### PR TITLE
Fix telemetry build issues

### PR DESCRIPTION
**Pull request by rupesh-k**
Verified telemetry fixes.

### CODE REVIEW

File: scripts/common/sonic-telemetry-build/build.sh [hunk 1]
```diff
@@ -12,6 +12,12 @@ sudo sed -ri 's/^unixsocketperm .../unixsocketperm 777/' /etc/redis/redis.conf
 sudo sed -ri 's/redis-server.sock/redis.sock/' /etc/redis/redis.conf
 sudo service redis-server start
 
+# Install libyang
+sudo dpkg -i buildimage/target/debs/stretch/libyang*.deb
+
+# Clone sonic-mgmt-framework repository
+git clone https://github.com/Azure/sonic-mgmt-framework
```
Comment (qiluo-msft) [inline scripts/common/sonic-telemetry-build/build.sh]: How is it used?

Comment (anand-kumar-subramanian) [reply scripts/common/sonic-telemetry-build/build.sh]: libyang is for validation of the configuration as described in https://github.com/Azure/SONiC/blob/master/doc/mgmt/Management%20Framework.md#3228-config-validation-library-cvl

Comment (qiluo-msft) [reply scripts/common/sonic-telemetry-build/build.sh]: I see `sonic-mgmt-framework` repo not used

Comment (anand-kumar-subramanian) [reply scripts/common/sonic-telemetry-build/build.sh]: Code inside mgmt-framework is shared with telemetry repo. telemetry repo will not compile if the mgmt-framework is not there.

Comment (qiluo-msft) [reply scripts/common/sonic-telemetry-build/build.sh]: `sonic-telemetry` repo is not self-contained, and this dependency is not mentioned in [doc](https://github.com/Azure/sonic-telemetry/#installing). Please consider improve the source code management, for example using git submodule, etc.

I will approve it to unblock.

**Comment by prsunny** [comment]
This is breaking 'bmtor' branch as there is no sonic-mgmt-framework taken into bmtor.. Seems like wrong dependency to me.  [build_log](https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/buildimage-mlnx-bmtor/54/)

**Comment by rupesh-k** [comment]
I see below ygot related errors in build.log  (captured in the end)
I guess these are because of the external ygot community changes. We might need similar fix as in below PR in telemetry repo.

https://github.com/Azure/sonic-mgmt-framework/pull/43

And also I see build.sh is not invoked from the bmtor/Jenkinsfile, so the changes in this commit may not have impact on this build failure.

Thanks

10:13:24  /tmp/go/src/github.com/openconfig/ygot/proto/yext/yext.pb.go:44:11: undefined: "github.com/golang/protobuf/proto".ProtoPackageIsVersion4
10:13:24  # github.com/openconfig/ygot/proto/ywrapper
10:13:24  /tmp/go/src/github.com/openconfig/ygot/proto/ywrapper/ywrapper.pb.go:46:11: undefined: "github.com/golang/protobuf/proto".ProtoPackageIsVersion4

**Pull request merged**